config.js

```diff
@@ -135,34 +135,35 @@ config.iota = {
 
     /**
      * Configuration for the IoT Manager. If the IoT Agent is part of a configuration composed of multiple IoTAgents
-     * coordinated by an IoT Manager, this section defines the information that will be used to connect with that manager.
+     * coordinated by an IoT Manager, this section defines the information that will be used to connect with that
+     * manager.
      */
-    iotManager: {
-        /**
-         * Host where the IoT Manager is located.
-         */
-        host: 'localhost',
+    //iotManager: {
```

Comment:
I understand the lines commented with `//` were some test but they should be changed back.

Reply:
Fixed: 333f0ec